src/apps/backend/modules/share-task-request/internal/store/share-task-request-db.ts

```diff
@@ -0,0 +1,54 @@
+import { Schema, Types } from 'mongoose';
+import { ShareTaskRequestStatus } from '../../types';
+
+export interface ShareTaskRequestDB  {
+  _id: Types.ObjectId;
+  task: Types.ObjectId;
+  account: Types.ObjectId;
+  status: ShareTaskRequestStatus;
+  sharedTask: boolean;
+  active: boolean;
+}
+
+export const ShareTaskRequestDBSchema: Schema = new Schema<ShareTaskRequestDB>(
+  {
+    task: {
+      type: Schema.Types.ObjectId,
+      ref: 'Task',
+      index: true,
+      required: true,
+    },
+
+    account: {
+      type: Schema.Types.ObjectId,
+      ref: 'Account',
+      index: true,
+      required: true,
+    },
+
+    status: {
+      type: String,
+      enum: ShareTaskRequestStatus,
+      default: ShareTaskRequestStatus.ACCEPTED,
+      required: true,
+    },
+
+    sharedTask: {
+      type: Boolean,
+      default: true, 
+    },
+    
```

Comment:
Why do we need this `sharedTask` field here? We already have `shared-task-requests` collection to get all shared tasks.